geany.appdata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<component>
+  <id type="desktop">geany.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <pkgname>geany</pkgname>
+  <name>Geany Integrated Development Environment</name>
```

Comment:
IMO, "Geany" , or "Geany IDE" if you really don't like a name only, would be better.  "Geany Integrated Development Environment" sounds a lot harder to read than needed.